docs/design/adrs/2024-03-18-use-cypres-as-a-framework-for-e2e-testing.md

```diff
@@ -1,17 +1,17 @@
 # Use Cypress as a framework for E2E testing
 
-* **Status:** proposed
+* **Status:** approved
 * **Proposer:** @superhindupur
```

Comment:
Sorry for the many changes in this file, they're all whitespace changes apart from marking the ADR as approved